src/main/java/org/spongepowered/server/SpongeVanilla.java

```diff
@@ -189,6 +191,7 @@ private static void start(String[] args) {
 
         // Register Minecraft plugin container
         MinecraftPluginContainer.register();
+        SpongeCommonContainer.register();
```

Comment:
If we pass the `MetadataContainer` as parameter into this method we can avoid calling `MetadataContainer.load()` twice, which loads and parses the metadata from the JAR.